zk/src/archive/web/js/zk/au.js

```diff
@@ -87,6 +87,13 @@ Copyright (C) 2008 Potix Corporation. All Rights Reserved.
 					return;
 				}
 
+				if (cmd == 'setAttr' && !zAu.doAfterProcessWgts) {
+					zAu.doAfterProcessWgts = [];
+				}
+				if (zAu.doAfterProcessWgts) {
```

Comment:
once doAfterProcessWgts is set, other wgts will be add to the array no matter 'setAttr' is called

Reply:
if (cmd == 'setAttr' && zAu.doAfterProcessWgts) {